src/write-results-to-db.js

```diff
@@ -0,0 +1,110 @@
+const knex = require("knex")
+const moment = require("moment-timezone")
+const config = require("./config")
+
+const writeResultsToDatabase = (results, { realtime } = {}) => {
+  const db = knex({ client: "pg", connection: config.postgres })
+
+  if (realtime) {
+    return _writeRealtimeResults({ db, results }).then(() => db.destroy())
+  } else if (results.query.dimensions.match(/ga:date/)) {
+    return _writeRegularResults({ db, results }).then(() => db.destroy())
```

Comment:
We use the `realtime` prop on the report to make the distinction between the realtime API and the Core Reporting API. This conditional is there to make sure that data we fetch from the Core Reporting API has an associated date.